apps/examples/src/examples/custom-components/README.md

```diff
@@ -3,6 +3,33 @@ title: Canvas components
 component: ./CustomComponentsExample.tsx
 category: ui
 priority: 2
+keywords:
```

Comment:
i've been saying to remove the custom API terms but _maybe_ this is the one exception :)